packages/superset-ui-legacy-preset-chart-big-number/src/BigNumber/transformProps.ts

```diff
@@ -0,0 +1,132 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import * as color from 'd3-color';
+import { getNumberFormatter, NumberFormats } from '@superset-ui/number-format';
+import { ChartProps } from '@superset-ui/chart';
+import getTimeFormatterForGranularity from '../utils/getTimeFormatterForGranularity';
+
+const TIME_COLUMN = '__timestamp';
+
+export default function transformProps(chartProps: ChartProps) {
+  const { width, height, formData, queryData } = chartProps;
+  const {
+    colorPicker,
+    compareLag: compareLagInput,
+    compareSuffix = '',
+    headerFontSize,
+    metric,
+    showTrendLine,
+    startYAxisAtZero,
+    subheader = '',
+    subheaderFontSize,
+    timeGrainSqla: granularity,
+    vizType,
+    timeRangeFixed = false,
```

Comment:
Is this the old `useFixedTimeRange` field?

Reply:
Yes